core/perf_test/PerfTest_ViewCopy_Raw.cpp

```diff
@@ -14,7 +14,7 @@
 //
 //@HEADER
 
-#include <PerfTest_ViewCopy.hpp>
+#include "PerfTest_ViewCopy.hpp"
```

Comment:
Why did you change?  Same comment everywhere below.

Reply:
I was getting:
```
error: 'PerfTest_ViewCopy.hpp' file not found with <angled> include; use "quotes" instead
```
 This has worked before because of `  KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})` in the respective `CMakeLists.txt` but I don't see any reason to use that. With the header in the same directory as the `.cpp` file, double quotes work just fine.

Reply:
Mmm actually that file would be in `CMAKE_CURRENT_SOURCE_DIR`.  I feel like we should do `target_include_directories(... ${CMAKE_CURRENT_SOURCE_DIR})` instead

Reply:
True about the `CMAKE_CURRENT_SOURCE_DIR` :ok_hand: - it's there in the subsequent line:
https://github.com/kokkos/kokkos/blob/d9d43bf62998854617e9224b074775aa1e6fa1ba/core/perf_test/CMakeLists.txt#L61-L62
So this stays intact and `PerformanceTest_*` can continue to work as they were.
Are you ok with NOT adding `target_include_directories(... ${CMAKE_CURRENT_SOURCE_DIR})` for benchmarks? (is there are reason to add that apart from using angled brackets?)